ReSharper.FSharp/test/data/features/quickFixes/removeUnusedNamedPattern/Unused union case named pattern 1.fs.gold

```diff
@@ -0,0 +1,3 @@
+type R = | R of a: int
+match R(0) with
+| _{caret} -> ()
```

Comment:
This changes the code meaning if when there're other branches or union cases. This is the case we've discussed previously. You should not remove the whole union case pattern in this case, only its fields.

The value of `x` is going to be different here after this quick fix is applied:
```fsharp
type U =
    | A of i: int
    | B

let x =
    match A(1) with
    | A(i = i) -> 1
    | B -> 2
```

Reply:
Ok I see so in this case this case after applying the quick fix the result would be : 
```fsharp
let x =
    match B with
    | A  _ -> 1
    | B -> 2
```